### ISSUE TITLE
[ISSUE] Failed to install

### ISSUE THREAD

**Issue by Th3N3rdyGam3r** [Styr1x/Browsingway#12]
I'm getting a failed to install error when trying to install Browsingway. I've tried restarting the game and it still doesn't work. Other mods are able to be installed and used without any issues. Let me know if there's anywhere I can get logs from or any other information you need.

**Comment by Styr1x** [comment]
Could you send me the dalamud.log file? Its located here: `%APPDATA%\XIVLauncher\dalamud.log`

**Comment by Th3N3rdyGam3r** [comment]
Here's some of the latest content from dalamud.log: [https://pastebin.com/pwZAeYwx](https://pastebin.com/pwZAeYwx)
I took the last consecutive session due to Pastebin's size limit, but let me know if you need more. The last failed install starts at line 526.

**Comment by TibiIius** [comment]
Just here to say that I do face the same problem. 
I'll leave [my log file](https://pastebin.com/1QTBvzYK) here aswell (can't check the author's pastebin right now, so I don't know if it's the same error). 

Side note: I do run Linux, I don't know if that could be the problem.

**Comment by Th3N3rdyGam3r** [comment]
Not sure why Pastebin put my log under moderation, so maybe this will work instead: [https://zerobin.net/?abded6af85886d23#ud7Y/FhzW4PtvH7LAb/ZLskdkqFzpYngfyvYZ4mokA0=](https://zerobin.net/?abded6af85886d23#ud7Y/FhzW4PtvH7LAb/ZLskdkqFzpYngfyvYZ4mokA0=)
I use Windows with Dalamud's default settings afaik.

**Comment by Th3N3rdyGam3r** [comment]
> Just here to say that I do face the same problem. I'll leave [my log file](https://pastebin.com/1QTBvzYK) here aswell (can't check the author's pastebin right now, so I don't know if it's the same error).
> 
> Side note: I do run Linux, I don't know if that could be the problem.
> 
> Edit: I also use legacy support to initialize Dalamud (crashes for me otherwise) if that is important.

Your pastebin is also giving me the same error

**Comment by Th3N3rdyGam3r** [comment]
Not sure what the issue was, but the game just launched and asked me to install an additional dependency for the plugin before saying the plugin is now installed.

**Comment by Styr1x** [comment]
Are you by any chance on Linux with Wine?
Last time I've seen the "Could not determine plugin directory" error it was on Linux. I already have a fix for that for the new version. If you aren't on Linux I can backport it to the current branch, but if you are I'd wait until 1.1 as the current version has other issues with Linux, too.

**Comment by Th3N3rdyGam3r** [comment]
No, I'm running Windows. I do have my documents directory on the D drive though

**Comment by Styr1x** [comment]
Ok, the document path shouldn't matter though. I'll push out 1.0.0.4 with the fix backported from 1.1 later, that should hopefully resolve your issue.

**Comment by Styr1x** [comment]
@Th3N3rdyGam3r @TibiIius 1.0.0.3 is out now, can you update ingame and check if it fixes your problem?

**Comment by Styr1x** [comment]
I plan to fix the bitmap render mode und may add CefSharp version selection, this might make it workable with linux. You can create new issues for linux and I will try to fix them as long as it doesn't compromise the windows user experience.

You could also take a look at kaminariss Next UI plugin (https://gitlab.com/kaminariss/nextui-plugin). It works with linux according to the XIVLauncher Discord. Next UI is more than just overlays, but overlays are one of its features that can be used without the others. Both plugins differ designwise as NUI loads Cef inside the FFXIV process itself, which makes linux support easier but tradeoffs that for stability (one of the key features of Browsingway is that it should never crash your game).

In short: please create a new issue, I'll tag it with linux and until it runs under linux take a look at NUI :)
